Packages/com.unity.inputsystem/InputSystem/Actions/InputActionState.cs

```diff
@@ -1179,7 +1182,7 @@ private static bool ShouldIgnoreControlStateChangeOnCompositeBinding(BindingStat
         /// from all controls. However, it would not necessarily result in more useful conflict resolution and would
         /// at the same time be much more expensive.
         /// </remarks>
-        private bool ShouldIgnoreControlStateChange(ref TriggerState trigger, int actionIndex)
+        private bool IsConflictingInput(ref TriggerState trigger, int actionIndex)
```

Comment:
Its a bit confusing that a Is-prefixed function updates state so maybe call this function something else since it does checking and modifies state?